sematic/examples/hackernews_summarization/__main__.py

```diff
@@ -0,0 +1,114 @@
+# Standard Library
+import argparse
+
+# Sematic
+from sematic.examples.hackernews_summarization.pipeline import (
+    CohereConfig,
+    HNFetchConfig,
+    OpenAIConfig,
+    pipeline,
+)
+
+
+def main():
+    parser = argparse.ArgumentParser("Hacker News Summarization")
+    parser.add_argument(
+        "--query",
+        action="store",
+        help="Search query to filter stories",
+        required=True,
+    )
+
+    parser.add_argument(
+        "--past-n-days",
+        type=int,
+        action="store",
+        default=1,
+        help="Past N number of days to fetch Hacker News for",
+        required=False,
+    )
+
+    parser.add_argument(
+        "--max-stories",
+        type=int,
+        action="store",
+        default=10,
+        help="Number of stories you'd want to fetch from HN",
+        required=False,
+    )
+
+    parser.add_argument(
+        "--cohere-model",
+        type=str,
+        action="store",
+        default="command",
+        help="Model you want to use for text summarization.",
+        required=False,
+    )
+
+    parser.add_argument(
+        "--cohere-api-key",
+        type=str,
+        action="store",
+        help="API key for cohere",
+        required=False,
+    )
+
+    parser.add_argument(
+        "--cohere-max-tokens",
+        type=int,
+        action="store",
+        default=100,
+        help="Max number of tokens",
+        required=False,
+    )
+
+    parser.add_argument(
+        "--openai-model",
+        type=str,
+        action="store",
+        default="text-ada-001",
+        help="Model you want to use for text summarization.",
+        required=False,
+    )
+
+    parser.add_argument(
+        "--openai-api-key",
+        type=str,
+        action="store",
+        help="API key for openai",
+        required=False,
+    )
+
+    parser.add_argument(
+        "--openai-max-tokens",
```

Comment:
Is it necessary to have 2 different arguments for the maximum number of tokens? Please unify into one.